sklearn/utils/validation.py

```diff
@@ -8,35 +8,18 @@
 # License: BSD 3 clause
 import warnings
 import numbers
+from inspect import getargspec
 
 import numpy as np
 import scipy.sparse as sp
 
 from ..externals import six
 from ..utils.fixes import signature
+from ..exceptions import (DataConversionWarning, NonBLASDotWarning,
+                          NotFittedError)
```

Comment:
Maybe put an inline comment to state that this import is required for backward compatibility only.